### DISCUSSION TITLE
Oopen appropriate layout depending on namespace

### DISCUSSION THREAD

**Discussion by rafal-gorecki** [foxglove/community#914] [General]
Hello,
I am looking for a method to open the same layout for several robots. All data is preceded by a namespace unique to each robot, including `/tf` and `/tf_static` data. Is there any way to remap these topics and add a namespace?

**Comment by jtbandes** [comment]
You can use a [topic alias extension](https://docs.foxglove.dev/docs/visualization/extensions/api/extension-context#registertopicaliases) to remap topics as desired. However, there is not currently a way to control which TF topics are used by the 3D panel. So you may run into issues if the TF data for different robots includes the same frame names. You could also consider splitting up the topics into separate [devices](https://docs.foxglove.dev/docs/devices).

**Comment by rafal-gorecki** [comment]
Thank you for your support ;)
